WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -19,6 +19,8 @@
           </th>
         <% end %>
 
+        <!-- Extra column for displaying number of events registered for -->
+        <th></th>
```

Comment:
Maybe write something in this header, like "Total" or "Total events" or "Events"